helm-charts/sep-service/values.yaml

```diff
@@ -0,0 +1,134 @@
+fullName: anchor-platform
+namespace: sandbox
+service:
+   containerPort: 8080
+   servicePort: 8080
+   replicas: 1
+   type: NodePort
+   name: sep
+image:
+   repo: stellar
+   name: anchor-platform
+   tag: 022afab
+   pullPolicy: Always
+ingress:
+   labels:
+     app: appy
+     test: label_test
+   annotations:
+     kubernetes.io/ingress.class: "public"
+     cert-manager.io/cluster-issuer: "default"
```

Comment:
Those can be deployment specific. I think it would be best to avoid setting them here as ingress class names might differ between k8s clusters. Also not every cluster will have cert-manager installed.